src/pages/distroSettings/tabs/ProviderTab/types.ts

```diff
@@ -1,18 +1,31 @@
 import { Provider } from "gql/generated/types";
 
-interface StaticProviderFormState {
+export enum BuildType {
+  Import = "import",
+  Pull = "pull",
+}
+
+// TODO: Append type with additional provider options, e.g. type ProviderFormState = StaticProviderFormState | DockerProviderFormState
```

Comment:
[nit] I think this no longer applies in the way it's written here, could update or just remove.